### PR TITLE
Add set intersect function

### PR DESCRIPTION
**Pull request by lumodon**

### CODE REVIEW

File: src/setIntersect.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+export default function setIntersect( set1 = [], set2 = [] ){
+  if( set1 == null || set1 == null){
```
Comment (shakalee14) [inline src/setIntersect.js:2]: duplicate conditions on an or statement? did you mean : 
``` if( set1 == null || set2 == null) ```

File: src/setIntersect.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  return result
+}
+
+
+function findDuplicate( item, arr ){
+  for ( let i of arr ){
+    if( i.constructor === Array || typeof i === 'object' || typeof i === 'function'){
+      return 'No nested arrays or objects'
+    }
+    if( item === i ){ return true }
```
Comment (shakalee14) [inline src/setIntersect.js:33]: create a new line on 33 after the curly brace, and after true

**Review by shakalee14 — COMMENTED** [review]
few suggestions

File: src/setIntersect.js (hunk 1 above)
Comment (mantinone) [reply src/setIntersect.js:2]: Opps.  That's probably in every set file, too.  We'll fix all four of those in a later commit.

**Pull request merged**